Neos.Neos/Classes/Domain/Model/Domain.php

```diff
@@ -53,13 +53,6 @@ class Domain implements CacheAwareInterface
      */
     protected $port;
 
-    /**
-     * @var Site
-     * @ORM\ManyToOne(inversedBy="domains")
-     * @Flow\Validate(type="NotEmpty")
-     */
-    protected $site;
```

Comment:
how should the relation ship of 

domain entity -> site node aggregate

work? Currently in this wip the domain wouldnt know its site directly but one has to fetch all site node aggregates and look into them